### ISSUE TITLE
Links that open in a new tab/window have no cue for screen readers

### ISSUE THREAD

**Issue by briri** [CDLUC3/dmptool#160]
We currently display an icon that indicates that the link is going to open in a new tab/window but this icon is not visible to a screen reader. We should add some screen reader only text to address this.

**Comment by stephaniesimms** [comment]
not sure I can test this w/o a screen reader @briri
